pontoon/checks/utils/__init__.py

```diff
@@ -0,0 +1,48 @@
+from django.http import JsonResponse
+
+import compare
+import translate_toolkit
+
+
+def get_quality_checks(entity, locale, plural_form, string, ignore_warnings):
+    """
+    Main function that performs all quality checks from frameworks handled in Pontoon.
+
+    :arg pontoon.base.models.Entity entity: Source entity
+    :arg pontoon.base.models.Locale locale: Locale of a translation
+    :arg basestring string: a translation
+    :arg int plural_form: plural form of a translation
```

Comment:
Can you clarify what `plural_form` is used for? The way I understand it, it tells you if a string contains plural, which makes this comment a big confusing (across the files). Wouldn't `translation contains plural forms` be clearer? 

If that's the case, why is it an int?

Reply:
Pontoon stores a number of plural forms (that's why it's int -> I'll double check if that's handled correctly again).

Reply:
So, would it be "number of plural forms for translation" more correct? If that's the case, there's a -1 at some point that doesn't make a lot of sense.

Reply:
It's the CLDR plural form, represented with an integer:
https://github.com/mozilla/pontoon/blob/master/pontoon/base/models.py#L2359
https://github.com/mozilla/pontoon/blob/master/pontoon/base/static/js/translate.js#L2640

Reply:
OK, then it would definitely benefit from a clearer comment across the code. 

"numeric representation of the CLDR plural form for the translation"?

Reply:
Plurals logic is gettext-only at this point, right? So, not cldr? (Will get interesting with Android)